inference-engine/tests/functional/plugin/myriad/subgraph_tests/nonzero_broadcast.cpp

```diff
@@ -19,6 +21,10 @@ class NonZero_Broadcast : public testing::WithParamInterface<BroadcastExplicitTe
                           public LayerTestsUtils::LayerTestsCommon {
 protected:
     void SetUp() override {
+        SetRefMode(LayerTestsUtils::RefMode::CONSTANT_FOLDING);
+        configuration[VPU_CONFIG_KEY(DETECT_NETWORK_BATCH)] = CONFIG_VALUE(NO);
+        configuration[VPU_CONFIG_KEY(DISABLE_REORDER)] = CONFIG_VALUE(YES);
```

Comment:
Why do you need disable reorder?

Reply:
Removed

Reply:
Apparently, it is still needed. Myriad2 tests fail

Reply:
So comment that and do only in case of MY2 (if it's possible)

Reply:
The comment was added. But I found no way to split tests between MX and M2 in new tests infra

Reply:
You can do the same as for old infra - rely on environment variable

Reply:
Or even more, suppose tests not run on MY2 - but it's probably will break CI, so not that good idea

Reply:
Fixed. Added `CheckMyriadX` helper method relies on the environment variable. You can check it out in the last commit